### ISSUE TITLE
so,I can't set the training step by myself,right?

### ISSUE THREAD

**Issue by yt605155624** [vanhuyz/CycleGAN-TensorFlow#77]
Could you plese tell me when will the training step stops? I can't set the steps myself, will it stops ubtil the loss lower than some number,can you tell me what the number is?

**Comment by mbbuehler** [comment]
In model.CycleGAN#optimize you can see that the training stops after a fixed number of steps (100K steps).

Of course, you can modify this number. You also might have to adjust the criterion for the learning rate decay.
